.github/workflows/track_backlog.yml

```diff
@@ -0,0 +1,16 @@
+name: Track new issues in backlog
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "5 * * * *" # Run at 5 minutes past every hour
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Track all issues
+      uses: dhruvkb/issue-projector@0.0.1
```

Comment:
Can we make sure that things that are in the Active Sprint project don't get added to the backlog?